webpack/assets/javascripts/react_app/components/AutoComplete/AutoComplete.js

```diff
@@ -40,7 +40,7 @@ class AutoComplete extends React.Component {
     const { useKeyShortcuts, handleSearch } = this.props;
     const instance = this._typeahead.current.getInstance();
     const { ENTER, FWD_SLASH, BACK_SLASH } = KEYCODES;
-    const didEventCameFromInput = e.target.tagName === 'INPUT';
+    const didEventCameFromInput = e.target.tagName === 'INPUT' || e.target.tagName === 'TEXTAREA';
```

Comment:
thanks @tbrisker, missed that one !
now that we have two `tagNames`, I suggest destructing it like:
```
const { tagName } = e.target;
const didEventCameFromInput = tagName === 'INPUT' || tagName === 'TEXTAREA';
```

Just looks cleaner, thanks !

Reply:
Thanks, updated